mitxpro/settings.py

```diff
@@ -725,7 +725,7 @@ def get_all_config_keys():
 )
 DRIVE_WEBHOOK_RENEWAL_PERIOD_MINUTES = get_int(
     "DRIVE_WEBHOOK_RENEWAL_PERIOD_MINUTES",
-    30,
```

Comment:
How are we getting 23.5 hours from this?

Reply:
24 hour expiration for the webhook (indicated by `DRIVE_WEBHOOK_EXPIRATION_MINUTES`) minus this value (30 minutes up until this PR)